src/Raven.Server/Documents/DocumentPutAction.cs

```diff
@@ -115,6 +115,12 @@ public DocumentPutAction(DocumentsStorage documentsStorage, DocumentDatabase doc
 
                 var result = BuildChangeVectorAndResolveConflicts(context, id, lowerId, newEtag, document, changeVector, expectedChangeVector, flags, oldValue);
 
+                if (flags.Contain(DocumentFlags.FromClusterTransaction) && nonPersistentFlags.Contain(NonPersistentDocumentFlags.FromSmuggler))
+                {
+                    var index = ChangeVectorUtils.GetEtagById(context.LastDatabaseChangeVector, _documentDatabase.DatabaseGroupId);
+                    changeVector = $"RAFT:{index}-{_documentDatabase.DatabaseGroupId}";
```

Comment:
This is indeed bad, because two document will have the same change vector.

the RAFT here is confusing, it doesn't reflect the raft index of the cluster, rather it is an incremental etag for each PUT/DELETE command that was passed through the cluster transaction.
With the unique Id of the database group.